tests/suite/test_app_protect_integration.py

```diff
@@ -410,3 +406,78 @@ def test_ap_enable_true_policy_correct_uds(
         )
         delete_items_from_yaml(kube_apis, src_ing_yaml, test_namespace)
         assert_invalid_responses(response)
+
+    def test_ap_multi_sec_logs(
+        self, kube_apis, crd_ingress_controller_with_ap, appprotect_setup, test_namespace
+    ):
+        """
+        Test corresponding log entries with multiple log destinations (in this case, two syslog servers)
+        """
+        src_syslog_yaml = f"{TEST_DATA}/appprotect/syslog.yaml"
+        src_syslog2_yaml = f"{TEST_DATA}/appprotect/syslog2.yaml"
+        log_loc = f"/var/log/messages"
+
+        print("Create two syslog servers")
+        create_items_from_yaml(kube_apis, src_syslog_yaml, test_namespace)
+        create_items_from_yaml(kube_apis, src_syslog2_yaml, test_namespace)
+
+        syslog_ep = get_service_endpoint(kube_apis, "syslog-svc", test_namespace)
+        syslog2_ep = get_service_endpoint(kube_apis, "syslog2-svc", test_namespace)
+
+        syslog_pod = kube_apis.v1.list_namespaced_pod(test_namespace).items[-2].metadata.name
+        syslog2_pod = kube_apis.v1.list_namespaced_pod(test_namespace).items[-1].metadata.name
+
+        with open(src_ing_yaml) as f:
+            doc = yaml.safe_load(f)
+
+            doc["metadata"]["annotations"]["appprotect.f5.com/app-protect-policy"] = ap_policy
+            doc["metadata"]["annotations"]["appprotect.f5.com/app-protect-enable"] = "True"
+            doc["metadata"]["annotations"][
+                "appprotect.f5.com/app-protect-security-log-enable"
+            ] = "True"
+
+            # both lists need to be the same length, if one of the referenced configs is invalid/non-existent then no logconfs are applied.
+            doc["metadata"]["annotations"][
+                "appprotect.f5.com/app-protect-security-log"
+            ] = f"{test_namespace}/logconf,{test_namespace}/logconf"
```

Comment:
@vepatel Perhaps we should also test that implicit namespace works?
```suggestion
            ] = f"{test_namespace}/logconf,logconf"
```

Reply:
but I've deployed test resources in a separate namespace so need to reference by the same, nothing related to test goes to default.